exporter/exporter.go

```diff
@@ -23,12 +23,17 @@ type Exporter struct {
 
 // Opts holds new exporter options.
 type Opts struct {
-	DSN  string
-	Path string
-	Port int
-	Log  *logrus.Logger
+	CollStatsCollections []string
+	DSN                  string
+	Log                  *logrus.Logger
+	Path                 string
+	Port                 int
 }
 
+var (
+	errCannotHandleType = fmt.Errorf("don't know how to handle data type")
```

Comment:
you anyway wrap this error, so any reason to extract it as separate error?

Reply:
Linter and go errors for Go 1.13+